requirements.txt

```diff
@@ -0,0 +1,13 @@
+--index-url https://pypi.python.org/simple/
+
+# Unit test stuff
+pytest==3.2.2
+pytest-pep8==1.0.6
+
+# Deep Learning frameworks
+tensorflow==1.3.0
```

Comment:
I use 1.2.0 personally (since 1.3 would require that I update CUDNN). I guess we can support even lower versions for now?

Reply:
I'll put in 1.2.0. Generally I'd say we use the latest-known-to-work versions.